simulator/risc_v/t/unit_test.cpp

```diff
@@ -175,6 +175,16 @@ struct TestData {
         instr.execute();
         CHECK( instr.get_v_dst() == dst);
     }
+
+    void make_128test( std::string str)
+    {
+        RISCVInstr<T> instr( str, 0);
+        instr.set_v_src( src1, 0);
+        instr.set_v_src( src2, 1);
+        instr.execute();
+        CHECK( narrow_cast<uint64>( instr.get_v_dst()) == narrow_cast<uint64>( dst));
```

Comment:
I suggest to use `if constexpr ( std::is_same_v<T, uint128>)` here to reduce code duplication